website/docs/cairo-1/06-interacting-with-starknet/06-multicall.md

```diff
@@ -24,9 +24,10 @@ function = "increase_balance"
 inputs = [42]
 ```
 
-[//]: # (TODO #1908 is this still valid? Does it refer to old deploy or new deploy <through UDC>?)
 
-Protostar supports two types of transactions — `deploy` and `invoke`, which take similar arguments to [`deploy command`](/docs/cli-reference#deploy) and [`invoke command`](/docs/cli-reference#invoke).
+Protostar supports two types of transactions — `deploy` and `invoke`, which take similar arguments to [`deploy` command](/docs/cli-reference#deploy) and [`invoke` command](/docs/cli-reference#invoke).
```

Comment:
```suggestion
Protostar supports few types of transactions, including `deploy` and `invoke`, which take similar arguments to [`deploy` command](/docs/cli-reference#deploy) and [`invoke` command](/docs/cli-reference#invoke).
```

I think we shouldn't be saying that 2 transactions are supported if we also support declare and deploy_account.

Reply:
I rephrased it so it might be more clear now